### PR TITLE
Fix #1324 VaSelect dropdown position

### LINKED ISSUE #1324: Wrong select dropdown position.
**Vuestic-ui version:** 1.3.1

**What is the current behavior?**
изображение

### PR DESCRIPTION
**Pull request by RVitaly1978**
## Description
close: #1324 

Changes:
- [x] - add prop `anchorSelector` to `va-dropdown` component
- [x] - update `va-input` markup
- [x] - update `va-select` and `va-time-input`: set `anchorSelector`
- [x] - update styles
- [x] - update docs and demo examples

chrome-capture

chrome-capture (1)

chrome-capture (2)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/va-select/VaSelect.vue [hunk 1]
```diff
@@ -12,6 +12,8 @@
     keep-anchor-width
     boundary-body
     :stateful="false"
+    anchorSelector=".va-input-wrapper__input"
+    :offset="[0, 2]"
```
Comment (m0ksem) [inline packages/ui/src/components/va-select/VaSelect.vue:16]: image

Strange offset here.

But according to our design files in figma dropdown should be changed later anyway.

```suggestion
```

File: packages/ui/src/components/va-time-input/VaTimeInput.vue [hunk 2]
```diff
@@ -2,10 +2,11 @@
   <va-dropdown
     class="va-time-input"
     v-model="isOpenSync"
-    :offset="[0, 10]"
+    :offset="[0, 2]"
```
Comment (m0ksem) [inline packages/ui/src/components/va-time-input/VaTimeInput.vue:5]:
```suggestion
```

**Review by m0ksem — APPROVED** [review]
Fix that small suggestion in select and time input and merge.

File: packages/ui/src/components/va-select/VaSelect.vue (hunk 1 above)
Comment (RVitaly1978) [reply packages/ui/src/components/va-select/VaSelect.vue]: set 1px offset

File: packages/ui/src/components/va-time-input/VaTimeInput.vue (hunk 2 above)
Comment (RVitaly1978) [reply packages/ui/src/components/va-time-input/VaTimeInput.vue]: set 1px offset

**Pull request merged**